specification/service-map/resource-manager/readme.typescript.md

```diff
@@ -7,7 +7,7 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-servicemap"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-servicemap"
+  output-folder: "$(typescript-sdks-folder)/sdk/servicemap/arm-servicemap"
```

Comment:
should this be "service-map"

Reply:
couldn't find this package servicemap in other repos :( Will keep it consistent with the specs repo